package.json

```diff
@@ -88,16 +82,31 @@
     "lodash.mapvalues": "4.6.0",
     "lodash.upperfirst": "4.3.1",
     "markdown-loader": "3.0.0",
+    "moment": "2.22.2",
+    "moment-timezone": "0.5.21",
     "omit-empty": "0.4.1",
     "prettier": "1.14.2",
     "rcfile": "1.0.3",
+    "react": "16.4.2",
     "react-dom": "16.4.2",
+    "react-intl": "2.4.0",
+    "react-router-dom": "4.3.1",
     "react-value": "0.2.0",
     "storybook-readme": "4.0.0-beta1",
+    "styled-components": "3.3.3",
     "stylelint": "9.5.0",
     "stylelint-config-standard": "17.0.0",
     "stylelint-order": "0.8.1",
     "svg-url-loader": "2.3.2",
     "url-loader": "1.1.1"
+  },
+  "peerDependencies": {
+    "moment": "2.22.2",
```

Comment:
I would specify ranges here. Otherwise you'll get warnings a lot on the consumer's end. E.g. `"react-dom": "^15.6 || ^16.0",`

Reply:
Yes, we should have ranges. We can also have it like e.g.`"react": ">=16"`